bucketlist/src/main/kotlin/io/aexp/bucketlist/BucketListClient.kt

```diff
@@ -34,6 +35,12 @@ interface BucketListClient {
             Observable<PagedResponse<PullRequestCommit>>
 
     /**
+     * @return observable of PR diff response
+     */
+    fun getPrDiff(projectKey: String, repoSlug: String, prId: Long, contextLines: Int, whitespace: Boolean,
```

Comment:
These are params of the diff endpoint in the Stash REST API
https://developer.atlassian.com/static/rest/stash/3.11.3/stash-rest.html#idp1811216
I can add comments to the code clarifying their purpose based on the description in the API docs if you'd like